cmd/api/src/model/tokenstring_test.go

```diff
@@ -0,0 +1,521 @@
+package model
```

Comment:
Most of these tests should be proper unit tests, but some are internal tests. Any test in a `_test.go` file should be in the package `<package>_test` instead. This ensures you're only testing the public API.

Please move any internal tests to a separate file named `<package>_internal_test.go` as needed.